### PR TITLE
Fix make dist command

### PR DESCRIPTION
**Pull request by mrow4a**
CC @DeepDiver1975

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,9 +30,9 @@ bower_deps=
 
 occ=$(CURDIR)/../../occ
 private_key=$(HOME)/.owncloud/certificates/$(app_name).key
-certificate=$(HOME)/.owncloud/certificates/$(app_name).crt
-sign=$(occ) integrity:sign-app --privateKey="$(private_key)" --certificate="$(certificate)"
-sign_skip_msg="Skipping signing, either no key and certificate found in $(private_key) and $(certificate) or occ can not be found at $(occ)"
+certificate=$(HOME)/.owncloud/certificates/$(app_name).csr
```
Comment (DeepDiver1975) [inline Makefile]: csr? no - you need the crt

File: Makefile [hunk 2]
```diff
@@ -32,7 +32,7 @@ occ=$(CURDIR)/../../occ
 private_key=$(HOME)/.owncloud/certificates/$(app_name).key
 certificate=$(HOME)/.owncloud/certificates/$(app_name).crt
 sign=$(occ) integrity:sign-app --privateKey="$(private_key)" --certificate="$(certificate)"
-sign_skip_msg="Skipping signing, either no key and certificate found in $(private_key) and $(certificate) or occ can not be found at $(occ)"
+sign_skip_msg="Skipping signing, either no key (.key) and certificate (.crt) found in $(private_key) and $(certificate) or occ can not be found at $(occ)"
```
Comment (mrow4a) [inline Makefile]: I am running 
```
$ openssl req -nodes -newkey rsa:4096 -keyout /Users/mrow4a/.owncloud/certificates/files_paperhive.key -out /Users/mrow4a/.owncloud/certificates/files_paperhive.crt -subj "/CN=files_paperhive"
$ make dist
```
and still getting `App signature is invalid: App Certificate is not valid.`
any clue @DeepDiver1975 ?

Comment (DeepDiver1975) [reply Makefile]: did you ever sign your csr properly as documented?

Comment (mrow4a) [reply Makefile]: @DeepDiver1975 https://github.com/owncloud/appstore-issues/issues/137

**Comment by mrow4a** [comment]
Can we merge this?

**Comment by mrow4a** [comment]
@DeepDiver1975 I also added some fixes to info.xml

**Comment by mrow4a** [comment]
This can be merged now I think

**Comment by mrow4a** [comment]
Diagnostics app has been released including the cosmetic changes included in this PR, and using adjusted `make dist` command as in this PR

**Pull request merged**